cabot/cabotapp/tasks.py

```diff
@@ -25,17 +28,64 @@
 logger = logging.getLogger(__name__)
 
 
-@task(ignore_result=True)
-def run_status_check(check_or_id):
-    if not isinstance(check_or_id, models.StatusCheck):
-        check = models.StatusCheck.objects.get(id=check_or_id)
+def _classify_status_check(pk):
+    """
+    Maps the check to either normal or high priority based on the dict:
+      cabot.celery_queue_config.STATUS_CHECK_TO_QUEUE / METRIC_CHECK_TO_QUEUE
+    """
+    check = models.StatusCheck.objects.get(pk=pk)
+
+    # We move this in here to avoid a circular dependency import failure
+    from cabot.metricsapp.models.base import MetricsStatusCheckBase
```

Comment:
why is it a circular import? how hard would it be to fix the circular import instead of this